nmoscommon/auth/nmos_auth.py

```diff
@@ -76,14 +78,15 @@ def get_auth_server_metadata(auth_href):
     return None
 
 
-class RequiresAuth(object):
+class AuthMiddleware(object):
```

Comment:
FYI a name change here probably requires a major version bump as this will already be used by other services.